presto-main/src/main/java/com/facebook/presto/connector/informationSchema/InformationSchemaPageSourceProvider.java

```diff
@@ -66,11 +68,13 @@
 {
     private final Metadata metadata;
     private final AccessControl accessControl;
+    private final TransactionManager transactionManager;
 
-    public InformationSchemaPageSourceProvider(Metadata metadata, AccessControl accessControl)
+    public InformationSchemaPageSourceProvider(Metadata metadata, AccessControl accessControl, TransactionManager transactionManager)
```

Comment:
This change does't appear in https://github.com/prestosql/presto/pull/90/commits/7baabd1f658a6eccfeec61101f61cbaf3c972d13 .  -- also doesn't seem useful?